api/api.go

```diff
@@ -50,6 +50,10 @@ type BlockChainAPI struct {
 	transactions storage.TransactionIndexer
 	receipts     storage.ReceiptIndexer
 	accounts     storage.AccountIndexer
+	// The EVM height at which the last import started.
+	// Keeps track at which EVM height we start importing
+	// after restarts/deployments.
```

Comment:
I would change the comment and the var name a bit so it's more clear, something like:

// stores the height from which the indexing resumed since the last restart.
// This is needed for syncing status
indexingResumedHeight uint64

Reply:
Makes sense :+1: Updated in https://github.com/onflow/flow-evm-gateway/pull/180/commits/ef802118627c2484f2c48592b07cc08cb97bb235